sql/item_mods.sql

```diff
@@ -3598,7 +3598,7 @@ INSERT INTO `item_mods` VALUES (11145,1,25);
 INSERT INTO `item_mods` VALUES (11145,8,8);
 INSERT INTO `item_mods` VALUES (11145,25,8);
 INSERT INTO `item_mods` VALUES (11145,386,45);
-INSERT INTO `item_mods` VALUES (11145,429,52);
+INSERT INTO `item_mods` VALUES (11145,429,38);    -- Increases Footwork's attack modifier to +38/256
```

Comment:
modifier.h says this:
`FOOTWORK_ATT_BONUS        = 429, // TODO: Raises the attack bonus of Footwork. (Tantra Gaiters +2 raise 100/1024 to 152/1024)`
status.lua says this:
`MOD_FOOTWORK_ATT_BONUS        = 429 -- Raises the attack bonus of Footwork. (Tantra Gaiters +2 raise 100/1024 to 152/1024)`

Also doesn't appear to actually be implemented right now..
https://github.com/DarkstarProject/darkstar/search?utf8=%E2%9C%93&q=FOOTWORK_ATT_BONUS&type=Code

Seeing mods added to modifier.h that don't actually do anything makes me sad.